server/backend/src/users/users.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Controller,
+  ForbiddenException,
+  Get,
+  Inject,
+  Param,
+} from '@nestjs/common';
+import { NotFoundError } from '../errors.types';
+import { UsersService } from './users.service';
+
+export type GetGroupsOutput = {
+  id: string;
+  name: string;
+  cameras: {
+    id: string;
+    name: string;
+  }[];
+}[];
+
+@Controller('users')
+export class UsersController {
+  constructor(@Inject(UsersService) private usersService: UsersService) {}
+
```

Comment:
endpoint comment(?)

Reply:
I might add Swagger decorators as documentation for our endpoints rather than comments, but I agree with documenting the endpoints in code as well